fix if multiple periods

agcopenhaver opened the pull request:
would split incorrectly, return empty bbox list, and error out if filename contained periods (eg: Potsdam_2_10_RGB.6.11.jpg). This fixes that, as it uses splitext and basename from os.path

Commit:
fix if multiple periods

would split incorrectly, return empty bbox list, and error out if filename contained periods (eg: Potsdam_2_10_RGB.6.11.jpg). This fixes that, as it uses splitext and basename from os.path

agcopenhaver commented:
still an error somewhere

Commit:
clean up mess I made. fix ext change from basename

agcopenhaver commented:
> still an error somewhere

was caused by splitext, which includes periods in extension. should be g2g

adbeda commented:
This is awesome. It turns out that extracting the extension with `os.path.basenameos.path.basename(f)[-1][1:]` also return error on image names with no multiple periods. I tested it with `os.split.split(f)[-1]`, it seems to be working on both types of image names.

adbedada committed:
used "splitext" instead of "basename" 

It turns out that extracting the extension with os.path.basenameos.path.basename(f)[-1][1:] also return error on image names with no multiple periods. I tested it with os.path.splitext(f)[-1], it seems to be working on both types of image names. Also, added "." to the extensions.
